apps/web/src/data/code-dictionary.ts

```diff
@@ -0,0 +1,22 @@
+export const codesDict = {
+  "200": {
+    code: "200",
+    name: "OK",
+    status_code: "200",
```

Comment:
What is the difference between `code` and `status_code`?

As code convention, we use `camelCase` instead of `snake_case` (e.g. `statusCode`).

Reply:
sure